ci-operator/config/openshift-priv/telemeter/openshift-priv-telemeter-master.yaml

```diff
@@ -9,7 +9,7 @@ build_root:
     cluster: https://api.ci.openshift.org
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.14
```

Comment:
You should not modify `openshift-priv/telemeter` files but the `openshift/telemeter` ones. The changes will be propagated to `openshift-priv` automatically.